entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Wait for MySQL to be ready
+echo "Waiting for MySQL..."
+until nc -z db 3306; do
```

Comment:
Consider adding a timeout to the loop that waits for MySQL to be ready. This will prevent the script from waiting indefinitely if there is an issue with the database service.